en/best-layout-practices.md

```diff
@@ -71,96 +43,57 @@ This section provides design guidance for the horizontal layout of Components an
+**The Inputs group has a fixed height and is pinned left and right, while the other group has both the width and height fixed and is pinned right. These two groups are put in another group that brings the whole layout together, which is pinned left and right and has a fixed height. Thus, when the form needs to change width, the Inputs Group and the Input elements inside will resize correctly to account for more or less available space, while the Button and Link Group will always keep its size and size of its elements, making sure they always occupy the same space on the right end of the form.**
 
 <img class="responsive-img" src="images/many_horizontal_nofix_do2.png" srcset="images/many_horizontal_nofix_do2@2x.png 2x" />
 
 Alternatively, one may want to have a form with fixed width that is always centered in relation to the group or artboard it belongs to.
 
-**Both Inputs, as well as the Button and the Hyperlink, have fixed widths and heights and are also put in a common group that has a fixed width and height and is positioned equally far from the sides of its parent both vertically and horizontally. All elements will preserve their dimensions under a responsive scenario and the form as a whole will be centered both ways.**
+**Both Inputs, as well as the Button and the Hyperlink, have fixed widths and heights and are put in a common group that has a fixed width and height and is positioned equally far from the sides of its parent both vertically and horizontally. All elements will preserve their dimensions under a responsive scenario and the form as a whole will be centered both ways.**
 
 <img class="responsive-img" src="images/many_horizontal_fix_do.png" srcset="images/many_horizontal_fix_do@2x.png 2x" />
 
-### Default Horizontal Layouts With More Elements
-
-When a Component or Pattern is positioned the distance to the left and top borders of the group or artboard it belongs to should be respected, therefore make sure to pin it left and top.
-
-**Each of the elements is pinned left and top which will generate code that renders it so that the left and top margins are respected in relation to the parent. The widths and heights of each element have been fixed as well in order to make sure that they don't resize if the parent does.**
-
-<img class="responsive-img" src="images/many_horizontal_nofix_do2.png" srcset="images/many_horizontal_nofix_do2@2x.png 2x" />
-
 ## All Margins
```

Comment:
This whole section is obsolete. Maybe we should remove it and add a section on Justification & Alignment , where we can also mention how paddings/margins are determined.